tests/fixtures/apidoc/compute_resources_role.json

```diff
@@ -0,0 +1 @@
+katello.json
```

Comment:
Why is this read? I hope it's a symlink. And think i remember github was way better than that to show it.

Reply:
Yes, that is a symlink.